Cargo.toml

```diff
@@ -12,52 +12,44 @@ edition = "2018"
 publish = false
 
 [features]
-default = ["tikv_alloc/default"]
+default = ["default_alloc", "protobuf-codec"]
+default_alloc = ["tikv_alloc/default"]
 tcmalloc = ["tikv_alloc/tcmalloc"]
 jemalloc = ["tikv_alloc/jemalloc", "engine/jemalloc"]
 mimalloc = ["tikv_alloc/mimalloc"]
 portable = ["engine/portable"]
 sse = ["engine/sse"]
 mem-profiling = ["tikv_alloc/mem-profiling"]
-profiling = ["profiler/profiling"]
 failpoints = ["fail/failpoints"]
+prost-codec = [
```

Comment:
em, I see nearly all the workspaces use these, seem too verbose, but I also find no way to simply them.

Reply:
Since features are unions, enabling one should be sufficient for all dependencies.